packages/validators/src/date.validators.ts

```diff
@@ -69,3 +69,27 @@ export function isDate(): BalValidatorFn {
     return DateFns.isDate(value)
   }
 }
+
+/**
+ * Returns `true` if the given data is at the same or after the value date
+ *
+ * ```typescript
+ * BalValidators.isMinDate('2000-01-01')('2000-01-01') // true
+ * BalValidators.isMinDate('2000-01-02')('2000-01-01') // true
+ * BalValidators.isMinDate(new Date(2020, 0, 1))(new Date(2020, 0, 1)) // true
+ * ```
+ */
+export function isMinDate(date: Date | string): BalValidatorFn {
```

Comment:
please add tests for this new function in date.validator.spec.ts

Reply:
i would suggest to name isBeforeOrEquals, why isMin?